front/app/containers/Authentication/steps/EmailAndPasswordSignUp/index.tsx

```diff
@@ -159,7 +159,7 @@ const EmailAndPasswordSignUp = ({
             </Button>
           </Box>
         </form>
-        <Box mt="32px">
+        <Box mt="22px">
```

Comment:
When using margins/padding, we normally try to use a multiple of four (so it will match when we at some point introduce a spacing variable). Do you think this could be changed to either 20px or 24px?? :)

Reply:
OK sure, I wasn't aware of that rule. I'll see what looks best.